.travis.yml

```diff
@@ -11,14 +11,26 @@ sudo: false
 
 cache:
   directories:
-    - $HOME/.composer/cache/files
+    - $HOME/.composer/cache
```

Comment:
Ok wasn't sure about this, some repo use the files dir some just cache dir. Will revert this!